test/recipes/70-test_sslrecords.t

```diff
@@ -464,7 +464,7 @@ sub change_version
     my $proxy = shift;
 
     # We'll change a version after the initial version neg has taken place
-    if ($proxy->flight != 2) {
+    if ($proxy->flight != 1) {
```

Comment:
It's not immediately obvious to me why this change is needed, but it seems harmless enough...

Reply:
I hadn't updated this test at all in the original version of my PR. Test 13 *should* have failed when I made the record version change (previously we ignored the record version, now we don't). - but it didn't. At least it didn't on Linux, but it did on AppVeyor (which is how I found this). The problem was we were making the record version change too late. Due to race conditions etc (platform dependent) sometimes it failed, sometimes it didn't. Making this change caused the test to fail reliably on all platforms.